reo/src/data_manager.py

```diff
@@ -498,6 +595,18 @@ def _get_REopt_cost_curve(self, techs):
                 n_segments_max = max(n_segments, n_segments_max)
                 n_segments_list.append(n_segments)
 
+            # [az] Not sure if we need this or not, first line is updated from DatFileManager and the rest may have been removed in this version
```

Comment:
@Bill-Becker can you please check Alex's question here (do we need this `elif`) and delete the comment?

Reply:
@NLaws we do for `boiler` and `elecchl`, however, I just noticed that `absorpchl` was also recently added to that list and that nullified the absorption chiller `installed_cost` being accounted for the in the objective function. I'm working on a commit to get absorption chiller added back to the upper loop of techs to account for its `installed_cost`. I'll also remove this comment.

Reply:
The bug fix I'm referring to is in https://github.com/NREL/REopt_Lite_API/pull/175/commits/0c8007d2f7f825111c50a39be9b36ac22bc3907d. I then removed the question comment and replaced it with an explanation of non-costed technologies in https://github.com/NREL/REopt_Lite_API/pull/175/commits/0c8007d2f7f825111c50a39be9b36ac22bc3907d.